app/javascript/src/utils/index.js

```diff
@@ -1,3 +1,13 @@
 import * as R from "ramda";
 
-export const isPresent = R.pipe(R.either(R.isNil, R.isEmpty), R.not);
+const isPresent = R.pipe(R.either(R.isNil, R.isEmpty), R.not);
+
+const slug = str => str.toLowerCase().split(" ").join("-");
```

Comment:
So what if I give the string as `UX_Designer`?  My expectation is `ux-designer` for this case too. If this method doesn't work for such cases, then refactor accordingly.